src/base/base.c

```diff
@@ -890,6 +890,7 @@ void xdebug_base_minit(INIT_FUNC_ARGS)
 	xdebug_old_execute_internal = zend_execute_internal;
 	zend_execute_internal = xdebug_execute_internal;
 
+	XG_BASE(nanotime_init) = xdebug_get_nanotime_init();
```

Comment:
I think I would have not done an assignment here, instead, I would have added below line 896:

```

xdebug_nanotime_init();
```

And then in the `xdebug_nanotime_init` function just access the elements in `XG_BASE()`

Reply:
fixed